crates/bevy_ui/src/ui_node.rs

```diff
@@ -347,6 +347,20 @@ impl From<Vec2> for ScrollPosition {
     }
 }
 
+/// Determines which axes of a node are *sticky* during scrolling.
+///
+/// A **sticky** node maintains its position along the specified axes
+/// instead of moving with its scrolled parent content when [`ScrollPosition`] is applied.
+#[derive(Component, Debug, Clone, Default, Deref, DerefMut, Reflect)]
+#[reflect(Component, Default, Clone)]
+pub struct ScrollSticky(pub BVec2);
```

Comment:
`ScrollSticky` carries the wrong implications, I think. This should be called `IgnoreScroll` or something. And the description is misleading too, the node isn't sticky, it's fixed in place.

Reply:
True, I was focusing on sticky headers, but IgnoreScroll would be a lot better.

I will update PR.